frontend/src/scenes/data-management/definition/DefinitionView.tsx

```diff
@@ -0,0 +1,153 @@
+import './Definition.scss'
+import React from 'react'
+import clsx from 'clsx'
+import { Divider } from 'antd'
+import { SceneExport } from 'scenes/sceneTypes'
+import { PageHeader } from 'lib/components/PageHeader'
+import { EditableField } from 'lib/components/EditableField/EditableField'
+import { AvailableFeature } from '~/types'
+import { ObjectTags } from 'lib/components/ObjectTags/ObjectTags'
+import { useActions, useValues } from 'kea'
+import { userLogic } from 'scenes/userLogic'
+import { DefinitionPopup } from 'lib/components/DefinitionPopup/DefinitionPopup'
+import {
+    definitionLogic,
+    DefinitionLogicProps,
+    DefinitionPageMode,
+} from 'scenes/data-management/definition/definitionLogic'
+import { LemonButton } from 'lib/components/LemonButton'
+import { formatTimeFromNow } from 'lib/components/DefinitionPopup/utils'
+import { humanFriendlyNumber, Loading } from 'lib/utils'
+import { ThirtyDayQueryCountTitle, ThirtyDayVolumeTitle } from 'lib/components/DefinitionPopup/DefinitionPopupContents'
+import { EventDefinitionProperties } from 'scenes/data-management/events/EventDefinitionProperties'
+import { getPropertyLabel } from 'lib/components/PropertyKeyInfo'
+import { EventsTable } from 'scenes/events'
+
+export const scene: SceneExport = {
+    component: DefinitionView,
+    logic: definitionLogic,
+    paramsToProps: ({ params: { id } }): typeof definitionLogic['props'] => ({
+        id,
+    }),
+}
+
+export function DefinitionView(props: DefinitionLogicProps = {}): JSX.Element {
+    const logic = definitionLogic(props)
+    const { definition, definitionLoading, singular, mode, isEvent, backDetailUrl } = useValues(logic)
```

Comment:
When you click into an event and then click edit, if you back out of the page and try to go back in, it defaults to the editing state since the mode persists in this case. Is this intended effect or do we want it to return to regular view?